spectrocloud/resource_macros.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func resourceMacro() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: resourceMacrosUpdate,
+		CreateContext: resourceMacrosCreate,
 		ReadContext:   resourceMacrosRead,
 		UpdateContext: resourceMacrosUpdate,
 		DeleteContext: resourceMacrosDelete,
```

Comment:
```suggestion
		DeleteContext: resourceMacrosDelete,
		Description: "A resource for creating and managing service output variables  and macros."
```
This is the global description for the resource itself.